package.json

```diff
@@ -42,6 +42,16 @@
           "type": "string",
           "default": "php",
           "description": "Sets a custom location to the php executable"
+        },
+        "artisan.docker.enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "Should commands get executed using docker?"
+        },
+        "artisan.docker.command": {
+          "type": "string",
+          "default": null,
```

Comment:
Maybe the default value could be `"docker-compose exec app"`? That is assuming this is the most common command.

Reply:
“app” string was an example. This is not the default. Keep attribute as null is the best choice. Will force user write their own full custom commands to get container running.